dom-querying/main.js

```diff
@@ -0,0 +1,10 @@
+console.log('hello, world');
+
+console.log(document.querySelector('h1'));
+console.dir(document.location);
+console.log(document.querySelector('#explanation'));
+console.dir(document.location);
+console.log(document.querySelector('.hint'));
+console.dir(document.location);
```

Comment:
What instruction told you to do this? 😕

Reply:
![image](https://user-images.githubusercontent.com/117887912/208491496-1cc2eff7-b532-4b05-9018-50e7a65494b2.png)